### ISSUE TITLE
statestats gives incorrect results when minlabel > 1 

### ISSUE THREAD

**Issue by dmd** [bbfrederick/capcalc#4]
```
from capcalc.utils import statestats
statestats([1,2,2,2], 2, 1)[0]
```

Expected result:

```
array([[0., 1.],
       [0., 2.]])
```

Actual result:

```
array([[0., 0.],
       [0., 3.]])
```

**Comment by dmd** [comment]
I suspect the problem is the call to `statefilter`.

**Comment by bbfrederick** [comment]
Yup.  Statefilter didn't properly initialize the first returned value.  Fixed now.

**Comment by dmd** [comment]
Closed by e6c2fd8432aa85b3682625221fa740a3290b5ae7

**Comment by dmd** [comment]
No, this is still broken.
